src/server/databaseClasses.py

```diff
@@ -0,0 +1,48 @@
+class EmployeeWorkTime:
+    def __init__(self, full_name, hours, last_clock_in, last_clock_out):
+        self.full_name = full_name
+        self.hours = hours
+        self.last_clock_in = last_clock_in
+        self.last_clock_out = last_clock_out
+
+
+class Inventory:
+    def __init__(self, ID, name, cost, stock):
```

Comment:
In general try to avoid using capital case for variables, so I would use `id` and not `ID` here. Same for the other classes